frontend/src/pages/GraphPF/NodeDecorator.tsx

```diff
@@ -0,0 +1,38 @@
+import { Tooltip } from '@patternfly/react-core';
+import {
+  DEFAULT_DECORATOR_RADIUS,
+  Decorator,
+  Node,
+  TopologyQuadrant,
+  getDefaultShapeDecoratorCenter,
+  observer
+} from '@patternfly/react-topology';
+import { IconType } from 'config/Icons';
+import React from 'react';
+
+interface Props {
+  element: Node;
+  icon: IconType;
+  quadrant: TopologyQuadrant;
+  tooltip?: string;
+}
+
+const NodeDecoratorInner: React.FC<Props> = ({ element, quadrant, icon, tooltip }) => {
+  const { x, y } = getDefaultShapeDecoratorCenter(quadrant, element);
+  const decoratorRef = React.useRef<SVGAElement | null>(null);
+
+  return (
+    <Tooltip triggerRef={decoratorRef} content={!!tooltip ? tooltip : icon.text}>
```

Comment:
Content value can be simplified to: `content={tooltip ?? icon.text}`

Reply:
Thanks @ferhoyos !.